requirements.txt

```diff
@@ -17,5 +17,6 @@ sphinx<6.0
 sphinx_rtd_theme
 statsmodels
 tqdm
+wandb
```

Comment:
This is debatable, but I wonder if we really want to add `wandb` as a dependency. (And maybe this is just in `requirements.txt`, but not an actual dependency that would be in the `Conda`/`pip` packages, so correct me if I'm mistaken.) I thought this would an optional dependency, basically.

Reply:
If we make it an optional dependency, then we have to handle it like we handle "gridmap" in SKLL and check for `ImportError` etc.

Reply:
Looking at their dependencies, I don' think it's that bad. Personally, I am okay with adding it as a straight-up dependency given that it will make our work at ETS easier for sure. And, like you said, it doesn't seem to be a thing that would bother people outside ETS anyway.